documentation/asciidoc/accessories/camera.adoc

```diff
@@ -1 +1,3 @@
 include::camera/camera_hardware.adoc[]
+
+include::camera/hqcam_filter_removal.adoc[]
```

Comment:
Huh, looking at the git history of that file, it looks like @aallan moved the original content out into the separate include-file (back in August 2021), but then forgot to edit the main body-text to use that include. :shrug:

Reply:
I did wonder why it was duplicated.

Reply:
Doh!